main.py

```diff
@@ -20,27 +20,39 @@
     # State
     state = UserTermState()
     player_active = 1
-    wait_for_ready = True
+    state.wait_for_ready = True
 
     val = ""
```

Comment:
Because of the walrus operator on line 33 `val` doesn't need to be assigned here.

Reply:
I may be dense but I am not totally following your comment. I have changed things so much I dont think this is relevant anymore? Iono let me know if I need to think about this deeper.

Reply:
Yes the line has changed from 33 to 24, but basically this `while (val := term.inkey()) != "q":` line means you don't need to declare `val = ""` here.

Reply:
Yeah the walrus is now on line 24.

Reply:
okay makes sense, done!